dotCMS/src/main/java/com/dotmarketing/cms/urlmap/URLMapAPIImpl.java

```diff
@@ -347,4 +380,6 @@ public ContentTypeURLPattern getPatternChange() {
             return matches;
         }
     }
+
+    private static class NotFoundURLPatternException extends RuntimeException {}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Classes implementing Serializable should set a serialVersionUID](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4543718)